operator/pkg/central/values/translation/translation.go

```diff
@@ -198,15 +208,15 @@ func getCentralPersistenceValues(p *platform.Persistence, checker *pvcStateCheck
 	return &persistence, nil
 }
 
-func getCentralComponentValues(c *platform.CentralComponentSpec, checker *pvcStateChecker, obsoletePVC bool) (*translation.ValuesBuilder, error) {
+func getCentralComponentValues(c *platform.CentralComponentSpec, m *platform.GlobalMonitoring, checker *pvcStateChecker, obsoletePVC bool) (*translation.ValuesBuilder, error) {
 	cv := translation.NewValuesBuilder()
 
 	cv.AddChild(translation.ResourcesKey, translation.GetResources(c.Resources))
 	if c.DefaultTLSSecret != nil {
 		cv.SetMap("defaultTLS", map[string]interface{}{"reference": c.DefaultTLSSecret.Name})
 	}
 
-	cv.SetBoolValue("exposeMonitoring", c.Monitoring.IsEnabled())
+	cv.SetBoolValue("exposeMonitoring", c.Monitoring.IsEnabled() || m.IsOpenShiftMonitoringEnabled())
```

Comment:
With https://github.com/stackrox/stackrox/pull/6669 this is no longer necessary. Up to you if you want to include it here and then revert later, or leave it out directly.

Reply:
Let's remove it when it will mergerd

Reply:
As https://github.com/stackrox/stackrox/pull/6669 is merged I'll remove it